app/src/main/java/com/example/topshotviewer2/MainActivity.kt

```diff
@@ -33,12 +36,38 @@ class MainActivity : ComponentActivity() {
 @Composable
 fun TopShotApp() {
     TopShotViewer2Theme {
-        Scaffold() { padding ->
+        Scaffold(
+            bottomBar = { TopShotBottomNavigation() }
+        ) { padding ->
             PlayerList(Modifier.padding(padding))
         }
     }
 }
 
+@Composable
+fun TopShotBottomNavigation(modifier: Modifier = Modifier) {
+    BottomNavigation(modifier) {
+        BottomNavigationItem(selected = true, onClick = {},
+            icon = {
+                Icon(
+                    imageVector = Icons.Default.Person,
+                    contentDescription = null
+                )
+            },
+            label = { Text(stringResource(R.string.bottom_players)) }
+        )
+        BottomNavigationItem(selected = false, onClick = {},
+            icon = {
+                Icon(
+                    imageVector = Icons.Default.Star,
```

Comment:
This item is not built yet, so it might be better to have a toast says that this feature is not available